### PR TITLE
Bug 1802010 - After adding a shortcut, show a confirmation message

### PR DESCRIPTION
**Pull request by gitstart**
### What
Show a snack bar when a website is added to shortcut

### Issue video
[fnx-62-issue.webm](https://user-images.githubusercontent.com/35276272/224277226-f77b37ad-81da-4663-a1d6-26f5f4a86f0c.webm)

### Demo video after fix
[fnx-62-fix.webm](https://user-images.githubusercontent.com/35276272/224272629-29ae582d-9b3e-4cc2-9e0c-5bab7d1f3538.webm)

### Pull Request checklist
 - [x] Quality: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
 - [x] Tests: This PR includes thorough tests or an explanation of why it does not
 - [ ] Changelog: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
 - [ ] Accessibility: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features
### After merge
- Milestone: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- Breaking Changes: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

---
This code was written and reviewed by GitStart Community. Growing future engineers, one PR at a time.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1802010

### CODE REVIEW

File: focus-android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -481,6 +481,9 @@
     <!-- Snackbar action to immediately open the successfully downloaded file. -->
     <string name="download_snackbar_open">Open</string>
 
+    <!-- Snackbar that will be displayed after a website has been added to shortcut. -->
+    <string name="added_to_shortcut">Added to shortcuts</string>
```
Comment (MatthewTighe) [inline focus-android/app/src/main/res/values/strings.xml:485]: Let's update this string's ID and it's content to match Fenix:

`<string name="snackbar_added_to_shortcuts">Added to shortcuts!</string>`

File: focus-android/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -481,6 +481,9 @@
     <!-- Snackbar action to immediately open the successfully downloaded file. -->
     <string name="download_snackbar_open">Open</string>
 
+    <!-- Snackbar that will be displayed after a website has been added to shortcut. -->
```
Comment (MatthewTighe) [inline focus-android/app/src/main/res/values/strings.xml:484]: nit:
```suggestion
    <!-- Snackbar that will be displayed after a website has been added to shortcuts. -->
```

**Review by MatthewTighe — CHANGES_REQUESTED** [review]
Couple small things, but this is otherwise looking good! Thanks for the contribution!

File: focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt [hunk 3]
```diff
@@ -555,6 +556,12 @@ class BrowserFragment :
         )
     }
 
+    private fun showShortcutAddedSnackBar() {
+        FocusSnackbar.make(requireView())
+            .setText(requireContext().getString(R.string.added_to_shortcut))
```
Comment (MatthewTighe) [inline focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt:561]: I think we will hit a compile error unless this resource ID is updated

**Review by MatthewTighe — CHANGES_REQUESTED** [review]

**Comment by debojyoti452** [comment]
Hello @MatthewTighe,
Requested changes are pushed. ✔

**Review by MatthewTighe — APPROVED** [review]
LGTM! Thanks for your contribution!

**Comment by MatthewTighe** [comment]
bors try

**Comment by MatthewTighe** [comment]
bors retry

**Pull request merged**